Src/Support/Google.Apis.Auth/OAuth2/Flows/GoogleAuthorizationCodeFlow.cs

```diff
@@ -83,6 +85,14 @@ public GoogleAuthorizationCodeFlow(Initializer initializer)
             userDefinedQueryParams = initializer.UserDefinedQueryParams;
         }
 
+        /// <inheritdoc/>
```

Comment:
I'd add a (non-XML) comment here saying that we're reimplementing the interface to make sure we return another GoogleAuthorizationCodeFlow.